packages/linear/src/components/PRPopover.tsx

```diff
@@ -15,128 +14,100 @@ interface PRPopoverProps {
   className?: string
 }
 
-const PR_REFERENCE_REPLACE_KEY = '<!-- __PR_REFERENCE__ -->'
-
-const usePRsFromDescription = (issue: Issue) => {
-  const texts = issue?.description?.match(/\[.+]\(https:\/\/github\.com\/.+\/pull\/\d+.*\)/gim)
-
-  return useMemo(() => {
-    const map = new Map<string, { number: string; texts: Set<string> }>()
-    texts?.forEach((text) => {
-      const [_t, _p, url, number] = text.match(/(\()(https:\/\/github\.com\/.+\/pull\/(\d+))/i) || []
-      if (url) {
-        const item = map.get(url) || { number, texts: new Set<string>() }
-        item.texts.add(text)
-        map.set(url, item)
-      }
-    })
-
-    return map
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [texts?.toString()])
+const isPullRequest = (attachement: Attachment) =>
+  attachement.source?.type === 'github' && !!attachement.source?.pullRequestId
```

Comment:
nit: typo in `attachment`

```suggestion
const isPullRequest = ({ source }: Attachment) =>  source?.type === 'github' && !!source?.pullRequestId
```